### PR TITLE
Fixed #27856 -- Improved date subtraction SQL on PostgreSQL.

### PR DESCRIPTION
**Pull request by vytisb**
[#27856](https://code.djangoproject.com/ticket/27856)

### CODE REVIEW

File: django/db/backends/postgresql/operations.py [hunk 1]
```diff
@@ -253,7 +253,9 @@ def subtract_temporals(self, internal_type, lhs, rhs):
         if internal_type == 'DateField':
             lhs_sql, lhs_params = lhs
             rhs_sql, rhs_params = rhs
-            return "age(%s, %s)" % (lhs_sql, rhs_sql), lhs_params + rhs_params
+            # This can be replaced with make_interval() when only
```
Comment (charettes) [inline django/db/backends/postgresql/operations.py]: Might want to mention the whole `make_interval(...)` SQL it should be replaced with.

**Comment by charettes** [comment]
buildbot, test on oracle

**Comment by charettes** [comment]
buildbot, test on oracle.

**Comment by charettes** [comment]
Buildbot, test on Oracle.

**Comment by timgraham** [comment]
buildbot, test on oracle.

**Comment by vytisb** [comment]
Thanks for the idea. I've looked into performance of these queries.

Setup:
The model from ticket description, filled with 10k rows.
`q = DateModel.objects.annotate(diff=F('d1') - F('d2')).values_list('diff', flat=True)`

Results:
`age(%s, %s)`
```
In [3]: %timeit -n 100 z = list(q.all())
100 loops, best of 3: 54.5 ms per loop
```
Average query length as reported by `connection.queries`: 18ms

`(interval '1 day' * (%s - %s))`
```
In [3]: %timeit -n 100 z = list(q.all())
100 loops, best of 3: 39.5 ms per loop
```
Average query length: 8.6ms

`make_interval(days := %s - %s)`
```
In [3]: %timeit -n 100 z = list(q.all())
100 loops, best of 3: 39.6 ms per loop
```
Average query length: 9.8ms


Given the results, I'm not sure it's worth mentioning `make_interval()` in the comment at all.

**Pull request merged**